R/plotconfiguration-label.R

```diff
@@ -54,16 +54,12 @@ LabelConfiguration <- R6::R6Class(
       for (labelName in names(labels)) {
         label <- labels[[labelName]]
         if(!isOfType(label, "Label")) {
-          labels[[labelName]] <- asLabel(text = label, font = currentTheme$fonts[[labelName]])
+          label <- asLabel(text = label, font = currentTheme$fonts[[labelName]])
         }
-
-        # Check Chosen angle is available
-        availableAngles <- c(0,90,180,270)
-        if (!(labels[[labelName]]$font$angle %in% availableAngles)) {
-          labels[[labelName]]$font$angle <- availableAngles[which(abs(label$font$angle - availableAngles) == min(abs(label$font$angle - availableAngles)))][1]
-          warning("Angles other than 0, 90, 189 and 270 are not available for title, subtitles, caption and axis titles. Replacing by closest available value: ", label$font$angle,".")
```

Comment:
Since label is `NULL` at the beginning for most cases and was not updated by the if loop, `label$font$angle` was still `NULL` in the warning message.